test/base_test.go

```diff
@@ -0,0 +1,125 @@
+package test
```

Comment:
One hint, which has been very useful to us, is to define build tags.
Its handy for running just a specific subset of your test suites.

Reply:
Another suggestion for you guys to evaluate, would be moving
those tests from `/test` to some more specific directory, like
`/test/base` or `/test/e2e/base`.